### ISSUE TITLE
Design usability features

### ISSUE THREAD

**Issue by NCrashed** [standardsats/valet#1]
We have the following possible usage scenarios that we should consider for UX improvements:
1) Send sats from the on-chain channel to the fiat channel.
2) Pay an invoice only from fiat channels.
3) Pay an invoice only from non-fiat channels.

P.S. These scenarios were irrelevant for originally hosted channels as sats in them had the same value that in ordinary channels.

**Comment by somes731** [comment]
Agreed with them however I think the technology should be in priority right now.
